pkgs/tools/compression/zopfli/default.nix

```diff
@@ -34,6 +34,7 @@ stdenv.mkDerivation rec {
     '';
     platforms = platforms.unix;
     license = licenses.asl20;
+    mainProgram = "zopfli";
```

Comment:
Let's move this and the `zstd` one to a separate PR.

Reply:
Doesn't really matter, we can just merge this as is.